bcda/web/api.go

```diff
@@ -300,8 +303,9 @@ func validateRequest(r *http.Request) ([]string, *fhirmodels.OperationOutcome) {
 	// validate optional "_since" parameter
 	params, ok = r.URL.Query()["_since"]
 	if ok {
-		_, err := fhirutils.ParseDate(params[0])
-		if err != nil {
+		instantRegex := `([0-9]([0-9]([0-9][1-9]|[1-9]0)|[1-9]00)|[1-9]000)-(0[1-9]|1[0-2])-(0[1-9]|[1-2][0-9]|3[0-1])T([01][0-9]|2[0-3]):[0-5][0-9]:([0-5][0-9]|60)(\.[0-9]+)?(Z|(\+|-)((0[0-9]|1[0-3]):[0-5][0-9]|14:00))`
```

Comment:
Does it make sense to have a comment here with a link to the bulk fhir spec showing that we didn't just "make up" this regex?
https://www.hl7.org/fhir/datatypes.html#primitive